CMakeLists.txt

```diff
@@ -17,6 +18,11 @@ else()
   set(DEBUG_FLAGS -Wall)
 endif()
 
+option(SPDLOG_SETUP_CPPTOML_EXTERNAL "Use external CPPTOML library instead of bundled" OFF)
+if(SPDLOG_SETUP_CPPTOML_EXTERNAL)
+	configure_file(cpptoml_config.h.in "${CMAKE_CURRENT_BINARY_DIR}/include/spdlog_setup/details/third_party/cpptoml.h" @ONLY)
```

Comment:
Nitpick: Tab should be 2 spaces, since the file generally follows 2 spaces convention.